tg.js

```diff
@@ -1,821 +1,821 @@
-windows.store.premium.purchase Sigurado ka bang nais mong bumili ng {0} araw ng Premium?
-windows.store.premium.gift Are you sure you want to Gift '{0}' {1} days of Premium?
-windows.playerlist.ban Sigurado ka bang nais mong ipagbawal si {0} mula sa larong ito?
-windows.playerlist.flag Sigurado ka bang nais mong I-flag si {0}?
-windows.playerlist.unflag Sigurado ka bang nais mong mag-Unflag si {0}?
-windows.playerlist.punish Sigurado ka bang nais mong parusahin ang manlalarong {0}?
-item.view.open Buksan ang Model Viewer
-item.view.more Iba pang Impormasyon
-item.sell.cant Hindi mabenta ang kagamitan.
-item.sell.quick Sigurado ka bang ibebenta mo ito agad {0}?
-social.profile.stats Kabuoang statistika
-social.profile.class Statistika ng mga Klase
-social.profile.sales Mga Binebenta
-social.profile.sales.market Mga Binebenta sa Market
-social.profile.skins Mga Skins o Disenyo [{0}]
-social.profile.skins.none Walang inakdang mga Skins o Disenyo mula sa User
-social.profile.games Mga Laro [{0}]
-social.profile.games.none Walang inakdang Laro o Mapa mula sa User
-social.profile.assets Mga Assets [{0}]
-social.profile.assets.none Walang inakdang Assets mula sa User
-social.profile.mods Mga Mods [{0}]
-social.profile.mods.none Walang inakdang Mods mula sa User
-social.search Resulta ng Paghahanap
-social.market.stats.rare Bihirang Mahanap na Items
-social.market.stats.circulation Nasa Sirkulasyon
-social.market.stats.units Bilang ng item na ito pandaigdig
-social.trades.in Papasok
-social.trades.in.head Pumapasok na Pag-aalok ({0})
-social.trades.out Palabas
-social.trades.out.head Palabas na Pag-aalok ({0})
-social.trades.history Kasaysayan
-social.trades.history.head Kasaysayan ng Pag-kakalakalan ({0})
-social.trades.decline Tanggihan Lahat
-social.trades.message Payagan ang mga Trade Message at Pag-alok sa Settings
-social.trades.warning Wala kang makukuhang kagamitan sa Pag-alok na ito!
-social.trades.accept Nais mong tanggapin ang Pag-alok na ito?
-stats.airdrops KR-Packages
-settings.quality.flapAnim Sniper Flap Animation
+settings.popup.header Mga Popups
+settings.popup.freeKRPop Ipakita ang Libreng KR Popup
+settings.experimental.header Experimental na Header
+settings.experimental.aimFreezeFix Ayusin ang Aim Freeze
+settings.experimental.mouseFlickFix Pag-ayos ng Mouse Flick
+settings.experimental.shaderRendering Pag-render
+settings.experimental.staticRendering Static na Rendering
+settings.client.header Kliyente
+settings.client.unlimitedFrames Unlimited na FPS
+settings.client.gameCap Record ng Laro
+settings.client.experimental Experimental Features
+settings.client.discordRPC RPC sa Discord
+settings.client.resourceSwapper Pagpalit ng Resources
+settings.client.clearCache I-clear ang Cache
+settings.client.resetSettings I-reset ang Settings ng Kliyent
+settings.client.acceleratedCanvas Accelerated ang Canvas
+settings.client.angleBackend ANGLE Graphics ang Backend
+settings.client.colorProfile Color ng Profile

```

Comment:
```suggestion
settings.client.colorProfile Profile ng Kulay
```
Current translation restranslates to "Profile color" and not "Color Profile"